tools/SourceKit/tools/sourcekitd-test/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ swift_add_public_tablegen_target(sourcekitdTestOptionsTableGen)
 add_sourcekit_executable(sourcekitd-test
   sourcekitd-test.cpp
   TestOptions.cpp
-  LLVM_LINK_COMPONENTS core option coverage lto
+  LLVM_LINK_COMPONENTS option coverage lto
```

Comment:
This broke late last year, maybe because of the LLVM monorepo transition or switching to CMake's built-in Swift support, so that this static library is linked twice. I'm able to reproduce on Arch linux x86_64 with the following build command, so I'm not sure why this doesn't show up on the linux CI:
```
> ./swift/utils/build-script -R --no-assertions --llvm-targets-to-build="X86;ARM;AArch64" -j9

> ./build/Ninja-Release/swift-linux-x86_64/bin/sourcekitd-test --help
: CommandLine Error: Option 'non-global-value-max-name-size' registered more than once!
LLVM ERROR: inconsistency in registered CommandLine options
```
I haven't dug into exactly why this happens, but can supply more info if someone is interested.

Reply:
@compnerd, you were the last one to modify this line and [you removed another LLVM library at the time](https://github.com/apple/swift/commit/717c4d3ebf9#diff-9d9699eccfb258b91e8fd3bda245ab61L16), so maybe you know what's going on here.

Reply:
Alright, separate pull forthcoming.

Reply:
Done, #29454.